ansible/playbook.yml

```diff
@@ -15,9 +16,10 @@
     - role: deerhide_users
     - role: svc_vault
 
-- name: Rasp's playbook
-  hosts:
-    - rasp
-  roles:
-    - role: svc_docker
-    - role: deerhide_users
+
+# - name: Rasp's playbook
+#   hosts:
+#     - rasp
+#   roles:
+#     - role: svc_docker
+#     - role: deerhide_users
```

Comment:
Why was this playbook commented out?

Reply:
First: we don't use it yet
Second: I just forgot to remove the comment on it